terraform/stacks/main/stack.tf

```diff
@@ -132,6 +132,10 @@ data "aws_sns_topic" "cg_notifications" {
   name = var.sns_name
 }
 
+data "aws_sns_topic" "cg_platform_notifications" {
```

Comment:
Is there any reason not to manage the SNS topic itself directly in Terraform?

Reply:
As far as I could tell we don't currently manage any other SNS topic through Terraform, so I was sticking with the current format

Reply:
Yeah. Despite the current state of the code, I'd like to manage our infrastructure through Terraform so that we know its provenance and can know the source of truth for its configuration. So I would like to add this SNS topic configuration to the code if it's not too much trouble

Reply:
I'll look into this and see how much work it might take